server/app/views/errors/NotFound.java

```diff
@@ -0,0 +1,76 @@
+package views.errors;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static j2html.TagCreator.a;
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.h1;
+import static j2html.TagCreator.p;
+import static j2html.TagCreator.span;
+
+import com.google.inject.Inject;
+import j2html.tags.specialized.DivTag;
+import j2html.tags.specialized.H1Tag;
+import play.i18n.Messages;
+import play.mvc.Http;
+import play.twirl.api.Content;
+import services.MessageKey;
+import views.BaseHtmlView;
+import views.HtmlBundle;
+import views.LanguageSelector;
+import views.applicant.ApplicantLayout;
+import views.style.BaseStyles;
+import views.style.ErrorStyles;
+import views.style.Styles;
+
+public class NotFound extends BaseHtmlView {
+
+  private final ApplicantLayout layout;
+  private final LanguageSelector languageSelector;
+
+  @Inject
+  public NotFound(ApplicantLayout layout, LanguageSelector languageSelector) {
+    this.layout = layout;
+    this.languageSelector = checkNotNull(languageSelector);
+  }
+
+  private H1Tag h1Content(Messages messages) {
+    return h1(
+            span(messages.at(MessageKey.ERROR_NOT_FOUND_TITLE.getKeyName())),
+            span(messages.at(MessageKey.GENERAL_SPACE.getKeyName())),
+            spanNowrap(messages.at(MessageKey.ERROR_NOT_FOUND_TITLE_END.getKeyName())))
```

Comment:
issue (blocking): instead of breaking this into multiple messages, create a single message that takes one or more arguments: https://www.playframework.com/documentation/2.8.x/JavaI18N#Formatting-messages

Reply:
I just got rid of the spanNowrap call. But I left the method in there in case someone else needs it. Otherwise I think its ready for another look